solution/ui/regulations/js/src/components/SimpleSpinner.vue

```diff
@@ -1,5 +1,6 @@
 <template>
     <div
+        :id="boxProperties"
```

Comment:
I removed most of the changes in here because I didn't want to make big changes to this component.  I just added a new size (`xs`) that we can use for this story and set it to have `0px` of `margin`.  That way no other implementations of this component are affected.